ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -325,6 +325,14 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted
+- as: e2e-ovirt
+  steps:
+    cluster_profile: ovirt
+    leases:
+    - count: 1
+      env: OVIRT_LEASED_RESOURCE
+      resource_type: ovirt-quota-slice
```

Comment:
Thanks for the catch, so I'll remove that section and use LEASED_RESOURCE instead :)
But on upgrade jobs we will need 2 leases so that means that we will need to add a count of 1 here right?

Reply:
Yeah, that sounds correct, provided you need two leases of the same type (I guess that's possible if these jobs simply need more resources).

Reply:
Yeap that is our case, hope to get more hardware soon and stop creating jobs with different resources but until then :)